CUDADataFormats/SiPixelDigi/interface/SiPixelDigisCUDASOAView.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CUDADataFormats_SiPixelDigi_interface_SiPixelDigisCUDASOAView_h
+#define CUDADataFormats_SiPixelDigi_interface_SiPixelDigisCUDASOAView_h
+
+#include <cuda_runtime.h>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+
+class SiPixelDigisCUDASOAView {
+public:
+  enum class StorageLocation { CLUS = 0, PDIGI = 2, RAWIDARR = 4, ADC = 6, XX = 7, YY = 8, MODULEIND = 9, MAX = 10 };
```

Comment:
http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1
2.17
```suggestion
  enum class StorageLocation { kCLUS = 0, kPDIGI = 2, kRAWIDARR = 4, kADC = 6, kXX = 7, kYY = 8, kMODULEIND = 9, kMAX = 10 };
```

Reply:
should comply now